src/EditorFeatures/CSharpTest/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -1121,70 +1135,66 @@ public async Task EventIsOnlyWritten()
             await TestDiagnosticMissingAsync(
 @"class MyClass
 {
-    private event System.EventHandler [|e|] { add { } remove { } }
+    private event System.EventHandler e { add { } remove { } }
     public void M()
     {
         // CS0079: The event 'MyClass.e' can only appear on the left hand side of += or -=
-        e = null;
+        {|CS0079:e|} = null;
     }
 }");
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)]
         public async Task FieldIsOnlyInitialized_NonConstant()
         {
-            await TestDiagnosticsAsync(
+            await TestDiagnosticMissingAsync(
 @"class MyClass
 {
-    private int [|_goo|] = M();
+    private int {|IDE0052:_goo|} = M();
     public static int M() => 0;
-}",
-    expected: Diagnostic("IDE0052"));
+}");
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)]
         public async Task FieldIsOnlyWritten_Deconstruction()
         {
-            await TestDiagnosticsAsync(
+            await TestDiagnosticMissingAsync(
```

Comment:
Probably rename this helper method to test `TestAnalyzerAsync` and the annotations determine if the test is verifying missing diagnostics or expected diagnostics without a code fix?

Reply:
➡️ Covered by https://github.com/dotnet/roslyn/pull/41755#discussion_r381076580